app.rb

```diff
@@ -0,0 +1,144 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'student'
+
+class App
+  attr_accessor :books, :people, :rentals, :teacher, :classroom, :student
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @teacher = []
+    @classroom = []
+    @student = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each_with_index do |book, _index|
+      puts "Title: '#{book.title}', Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    puts(@people.map { |person| "[#{person.type}] Name: #{person.name}, ID: #{person.id} Age: #{person.age}" })
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]:'
+    choice = gets.chomp.to_i
+    puts 'Age:'
+    age = gets.chomp.to_i
+    puts 'Name:'
+    name = gets.chomp
+
+    if choice == 1
+      person = create_student(age, name, parent_permission_get, parent_permission_get ? classroom_get : nil)
+    elsif choice == 2
+      person = create_teacher(age, name)
+    else
+      puts 'Invalid choice. Please enter 1 for student or 2 for teacher.'
+      return
+    end
+
+    if person
+      @people << person
+      puts 'Person created successfully'
+    else
+      puts 'Invalid choice. Please enter 1 for student or 2 for teacher.'
+    end
+  end
+
+  def create_student(age, name, parent_permission, classroom)
+    student = Student.new(age, classroom, name: name, parent_permission: parent_permission)
+    student.type = 'Student'
+    student
+  end
```

Comment:
Example video
![image](https://github.com/toyman640/school-library/assets/78251927/8ebe13b8-7cc2-48e3-9c95-4344191edd33)

According to the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), creating a student requires:
- Age
- Name
- Has parent permission? 

Your app 
![image](https://github.com/toyman640/school-library/assets/78251927/d483f1f4-d7c3-483c-bf69-51ea12c78267)

- Has parent permission? (please remove the duplicate)
- (Kindly remove the `Classroom label` so that your console app will behave in the same way as it is in the example.) 😊